R/estimateDiversity.R

```diff
@@ -70,8 +73,8 @@ NULL
 setGeneric("estimateDiversity",signature = c("x"),
            function(x, abund_values = "counts",
                     index = c("shannon","simpson","inv_simpson", "richness",
-                              "chao1", "ACE"),
-                    name = index, ...)
+                              "chao1", "ACE", "coverage"),
+                    name = index, threshold = NULL, ...)
```

Comment:
I think you should pick up the optional argument `threshold` later on

Reply:
Changed